content/well-architected-framework/docs/docs/automate-and-define-processes/create-high-performing-teams/collaboration.mdx

```diff
@@ -0,0 +1,30 @@
+---
+page_title: Collaboration
+description: Learn methods for team collaboration.
+---
+
+# Collaboration
+
+Collaboration amongst team members is important for the quality and quantity of work produced. Tools and processes can enable team collaboration.
```

Comment:
i think we can expand this to allude to the headings (git/peer review and team planning)

for next steps, we can link directly to communication too since collaboration and communication go hand in hand